core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,8 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
-    @media (max-width: 768px) {
-      .oppia-footer {
-        position: relative;
-      }
+    .oppia-wide-panel-content {
+      margin-bottom: 15em
```

Comment:
@llKCll Yes, you can push the commit. 
> Also, I have tried margin instead of bottom-margin as you suggested and appears to leave too much space above the error message.
Using 15em units so can lead to such results. I was suggesting that change in pixels (px) units.